docs/migration.md

```diff
@@ -0,0 +1,121 @@
+# Lisk V4.x Migration guide
+
+This section explains how to migrate a Lisk Core v3.x node to Lisk Core v4.x using the Lisk migrator.
+
+The Lisk migrator CLI tool will generate a new genesis block for Lisk Core v4.
+The new genesis block is created based on a snapshot of the old blockchain (running on Lisk Core v3.x) at a given height.
+
+> Note: All actively forging validators on the Lisk Mainnet and the Lisk Testnet need to follow this guide to correctly migrate their nodes to the new network, in order to not miss any blocks after the network hard fork.
+
+Please ensure you are running version 3.x of Lisk Core to be able to seamlessly migrate to Lisk Core 4.x.
+
+## Setting up the Lisk migrator
+
+### Download checksum and verify
+
+Download the checksum and verify the successful download of lisk-migrator.
+
+```
+curl -o lisk-migrator-v2.0.0.SHA256 https://downloads.lisk.com/lisk-migrator/lisk-migrator-v2.0.0.SHA256
+```
+
+### Extract and add to PATH
+
+```
+tar -xf ./lisk-migrator-v2.0.0.SHA256
+```
+
+Make the `lisk-migrator` command available in the PATH, e.g. by executing the following command:
+
+```
+export PATH="$PATH:$HOME/lisk-migrator/bin"
+```
+
+> Replace `$HOME` with the absolute path of where the `lisk-migrator` folder is located, in case it was extracted somewhere else other than in your home directory.
+
+> Alternatively migrator setup can be done by following the steps defined in section [here](../README.md).
+
+## Migration Steps
+
+**Check the announced snapshot height**
+
+- For Mainnet: `TBD`
+- For Testnet: `TBD`
+
+### Run Lisk Migrator
+
+The Lisk migrator v2.0.0 also allow users to download and start Lisk Core v4.x automatically after the migration. It can be done by passing the relavant flags.
```

Comment:
```suggestion
The Lisk Migrator v2 also allow users to download and start the Lisk Core v4.x automatically post migration. It can be done by passing the relevant flags.
```